src/assets/scss/application.scss

```diff
@@ -58,3 +64,6 @@ $govuk-assets-path: "/public/";
   @include govuk-responsive-margin(0, "bottom");
 }
 
+.govuk-date-input {
+  margin-bottom: -30px;
```

Comment:
Do you reckon we can use `govuk-spacing(-6)` here instead?

Reply:
Unfortunately the govuk spacing components don't allow negative values!